src/java.base/share/classes/java/lang/classfile/package-info.java

```diff
@@ -383,17 +397,23 @@
  * Such suppression may be beneficial when transformation removes many elements,
  * resulting in many unreferenced constant pool entries.
  *
- * <h3>Transformation handling of unknown classfile elements from a future</h3>
- * To achieve stability of transformations in future JDK releases it is desired
- * to set appropriate response to potential new classfile elements appearing.
- * Transformation designed to handle all situation should be implemented strict way,
- * so it fails when a new classfile element appears. As for example in the following
- * strict compatibility-checking code transformation snippet:
+ * <h3>Transformation handling of unknown classfile elements</h3>
+ * Future JDK releases may introduce new classfile elements, not known at the
+ * development time of a custom transformation code.
+ * To achieve deterministic stability of transformations running on future JDK
```

Comment:
This para seems largely redundant. Perhaps you can merge with the next? E.g. "To achieve deterministic stability, classfile transforms interested ..."